app/routes.py

```diff
@@ -1,2 +1,199 @@
-from flask import Blueprint
+from flask import Blueprint, request, make_response, jsonify
+from sqlalchemy.orm import query
+from sqlalchemy import desc
+from app.models.task import Task
+from app.models.goal import Goal
+from app import db
+from datetime import datetime
+# below for Slackbot
+import requests
+import os
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+# Get all tasks and create a task
+
+
+@tasks_bp.route("", methods=["GET", "POST"])
+def handle_tasks():
+    if request.method == "GET":
+        sort_order = request.args.get("sort")
+        if sort_order == "asc":
+            tasks = Task.query.order_by(Task.title)
+        elif sort_order == "desc":
+            tasks = Task.query.order_by(desc(Task.title))
+        else:
+            tasks = Task.query.all()
+
+        tasks_response = []
+        for task in tasks:
+            tasks_response.append({
+                "id": task.task_id,
+                "title": task.title,
+                "description": task.description,
+                "is_complete": False if not task.completed_at else True,
+            })
+        return jsonify(tasks_response)
+    elif request.method == "POST":
+        request_body = request.get_json()
+
+        # check for missing items
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return make_response({"details": "Invalid data"}, 400)
+
+        new_task = Task(title=request_body["title"],
+                        description=request_body["description"],
+                        completed_at=request_body["completed_at"])
+        db.session.add(new_task)
+        db.session.commit()
+
+        return make_response({
+            "task": {
+                "id": new_task.task_id,
+                "title": new_task.title,
+                "description": new_task.description,
+                "is_complete": False if not new_task.completed_at else True}}, 201)
```

Comment:
just like we did in the review session, think about how you could create a helper function to call here instead of writing this every time.